CMakeLists.txt

```diff
@@ -56,7 +56,8 @@ pkg_check_modules(
   pangocairo
   libdrm
   gbm
-  hyprutils>=0.2.3)
+  hyprutils>=0.2.3
+  sdbus-c++)
```

Comment:
you are using sdbus-c++ 2 here, so please add `>=2.0.0` to not confuse packagers (xdph still uses 1.x)

Reply:
What am I using from sdbus-c++>=2.0.0? On my system, I only have 1.5.0 installed.

Reply:
oh. I thought the syntax looked 2.0-like. Use 2.0 then. If you have a system 1.5, either install 2.0 or add a subproject and check it out to the latest release.

2.0 is breaking and I don't want to "fix it later", already lazy to do it with xdph

Reply:
Done.